src/BenchmarksServer/Startup.cs

```diff
@@ -262,7 +266,8 @@ private static void AddSourceDependencies(string path, string benchmarksDir, IEn
             benchmarksProjectDocument.Root.Add(netFrameworkReferences);
             benchmarksProjectDocument.Root.Add(netCoreReferences);
 
-            using (var writer = XmlWriter.Create(benchmarksProjectPath, new XmlWriterSettings { Indent = true, OmitXmlDeclaration = true }))
+            using (var stream = File.OpenWrite(benchmarksProjectPath))
+            using (var writer = XmlWriter.Create(stream, new XmlWriterSettings { Indent = true, OmitXmlDeclaration = true }))
```

Comment:
The `Create(string)` overloads only exist in .NET Framework and newer versions of `NETStandard.Library`.  We could go back to the `Create(string)` overload once we move to a newer version of `NETStandard.Library`.